### PR TITLE
Post workshop tutorial refresh

### PR DESCRIPTION
**Pull request by jsignell**
- [x] @jsignell: Expand or delete the classifier ensemble example
- [x] @jsignell: Set random seed in all notebooks that use random state
- [ ] @jsignell: Rationalize the division between user guide and topics
   - [ ] Delete topics that are now subsumed by tutorials
   - [ ] Boost remaining topics to be more in depth
   - [x] Reduce complexity and dataset size in tutorials
- [x] @jsignell: Simplify resampling and alignment, to allow details to be skipped
- [x] @jsignell: Remove datashader regridding? (Just use xarray)

### CODE REVIEW

File: examples/ai18.ipynb [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "   * **60 min** &nbsp;*Lunch break*<br><br>\n",
     "   \n",
     "- *Running ML or other analysis or simulation tools*\n",
-    "   * **30 min** &nbsp;[5 - Machine Learning](tutorial/05_Machine_Learning.ipynb): Specifying an ML pipeline using the prepared training data.\n",
+    "   * **30 min** &nbsp;[5 - Machine Learning](tutorial/04_Machine_Learning.ipynb): Specifying an ML pipeline using the prepared training data.\n",
```
Comment (jbednar) [inline examples/ai18.ipynb]: 5 - no longer matches 04_ , and the same below.  Probably better to go ahead and renumber things in this index to match how we'd give the tutorial if we do it again; the old version (should tag it!) can be retrieved if we want to know how we actually gave it.

File: examples/tutorial/01_Data_Ingestion.ipynb [hunk 2]
```diff
@@ -409,14 +380,15 @@
...
+    "landsat_5_xda = landsat_5.read()\n",
+    "type(landsat_5_xda)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Since `xarray` objects store raster information in a predictable way, we can even project the image onto lat lon coordinates using the `geo` key word argument."
+    "We can use tab completion to explore what other information is stored on our xarray.DataArray object. We can use tab completion to explore attributed and methods available on our object."
```
Comment (jbednar) [inline examples/tutorial/01_Data_Ingestion.ipynb]:
```suggestion
    "We can use tab completion to explore what other information is stored on our xarray.DataArray object. We can use tab completion to explore attributes and methods available on our object."
```

File: examples/tutorial/04_Machine_Learning.ipynb [hunk 3]
```diff
@@ -361,6 +321,13 @@
     "                         persist_embedding=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**This is the slow part.** Then we'll fit the model to out data `X`. This is the part that will take a noticeable amount of time. Something like 1 minute for the data in this tutorial or 9 minutes for a full size landsat image."
```
Comment (jbednar) [inline examples/tutorial/04_Machine_Learning.ipynb]:
```suggestion
    "**This is the slow part.** Then we'll fit the model to out data `X`. This is the part that will take a noticeable amount of time. Something like 1 minute for the data in this tutorial or 9 minutes for a full-size landsat image."
```

File: examples/tutorial/04_Machine_Learning.ipynb [hunk 4]
```diff
@@ -467,16 +425,7 @@
...
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "gts.ESRI * unstacked.hvplot(x='x', y='y', datashade=True, geo=True, height=500, cmap='Category10')"
+    "gv.tile_sources.ESRI * unstacked.hvplot(x='x', y='y', geo=True, height=500, cmap='Category10', alpha=0.7)"
```
Comment (jbednar) [inline examples/tutorial/04_Machine_Learning.ipynb]: I think ESRI is deprecated, and should be something more specific (e.g. EsriImagery?)

File: examples/tutorial/index.ipynb [hunk 5]
```diff
@@ -16,16 +16,28 @@
...
+    "5. [Data Visualization](./05_Data_Visualization.ipynb): How to visualize your data throughout the workflow, starting from data ingestion to the final machine learning product."
    ]
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
```
Comment (jbednar) [inline examples/tutorial/index.ipynb]: Need to call strip_kernel

File: examples/tutorial/03_Alignment_and_Preprocessing.ipynb [hunk 6]
```diff
@@ -221,23 +160,19 @@
...
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "ds.sel(x=x_center, y=y_center, method='nearest').compute()"
+    "### Select region of interest\n",
+    "\n",
+    "We'll define a central point in lat, lon convert that point to the *coordinate reference system (CRS)* of the data, and then use the area around the central point as the Region of Interest (ROI). \n",
+    "\n",
+    "The first step is getting the CRS. This information is stored in the attributed of our original landsat data. Let's take a look at it:"
```
Comment (jbednar) [inline examples/tutorial/03_Alignment_and_Preprocessing.ipynb]:
```suggestion
    "The first step is getting the CRS. This information is stored in the attributes of our original landsat data. Let's take a look at it:"
```

Comment (jsignell) [reply examples/tutorial/03_Alignment_and_Preprocessing.ipynb]: What the heck! I guess that shows how much I rely on spellcheck.

Comment (jbednar) [reply examples/tutorial/03_Alignment_and_Preprocessing.ipynb]: Unfortunately, s and d are right next to each other on the keyboard...

**Pull request merged**